utils/autotailor

```diff
@@ -52,21 +52,18 @@ def is_valid_xccdf_id(string):
         string) is not None
 
 
-class Tailoring:
+class Profile:
     def __init__(self):
-        self.id = "xccdf_auto_tailoring_default"
         self.reverse_dns = DEFAULT_REVERSE_DNS
-        self.version = 1
         self._profile_id = None
         self.extends = ""
-        self.original_ds_filename = ""
         self.profile_title = ""
 
-        self.value_changes = []
-        self.rules_to_select = []
-        self.rules_to_unselect = []
-        self.groups_to_select = []
-        self.groups_to_unselect = []
+        self.value_changes = set()
+        self.rules_to_select = set()
+        self.rules_to_unselect = set()
+        self.groups_to_select = set()
```

Comment:
Wouldn't the lists be better to ensure deterministic output? With this change, the generated profiles are ordered randomly. The order or elements shouldn't matter for the functionality but could be unpleasant for users running diffs or our tests. I suggest using lists or keep sets but sort the items at serializing.

Reply:
Sets are now converted to sorted lists before being rendered as XML.